### DISCUSSION TITLE
Everything is italics on Firefox

### DISCUSSION THREAD

**Discussion by matejdro** [FreshRSS/FreshRSS#7129] [Q&A] [answered]
### Describe the bug

Not sure if it was recent update, but suddenly the whole Fresh RSS interface is in italics for me:

![image](https://github.com/user-attachments/assets/af8db5d0-c958-4901-af52-b4c09e9041fb)

this includes the login screen:

![image](https://github.com/user-attachments/assets/68ba7e31-7d8b-4b42-b6fa-d557843fbe55)

### To Reproduce

1. Just open FreshRSS

### Expected behavior

Everything should not be in italics

### FreshRSS version

`freshrss/freshrss:edge image (aaebe80c327e)` (downloaded on 2024-12-18T08:30:38Z)

### Environment information

- Device: Desktop
- OS: Arch Linux
- Browser: Firefox 133.0.3, not happening on Chrome 131.0.6778.139 
- Database version: Postgres
- PHP version: One included in official docker image
- Installation type: Docker

### Additional context

It only seem to happen on Firefox on my desktop. Both Chrome on same machine and Firefox do not experience this. As far as I can tell, no other website is affected by this. It also happens in incognito mode with all my extensions disabled.

**Comment by Frenzie** [comment]
It could be worth trying ctrl+f5 or similar to force refresh all cached files. I'm not sure what might be the matter though.

**Comment by math-GH** [comment]
I have no clue what could be the root of this behavior. 

If you are able you could try to investigate it with the browser development tools

**Comment by Alkarex** [comment]
Check in the FreshRSS extensions. You may have some custom styles in UserCSS or CustomCSS

**Comment by matejdro** [comment, marked as answer]
Issue appeared to be my local setup. Maybe it was caused by a system update or something.

FreshRSS wants OpenSans font and for some reason Firefox was picking "OpenSans Italic" by default (even though OpenSans Regular was available). 

Ironically enough, the solution was to remove the system provided OpenSans font (by removing the https://archlinux.org/packages/extra/any/ttf-opensans/ package), which made Firefox fall back to its internal OpenSans font which works fine.

Sorry for bothering you and thanks a lot for your help anyway.

**Reply by Alkarex** [reply]
If you come back to your previous state with the local-OS font, could you please try https://github.com/FreshRSS/FreshRSS/pull/7215 ?

**Comment by AesaraB** [comment]
@math-GH @Alkarex 

If Open Sans refers to [this font](https://fonts.google.com/specimen/Open+Sans), then you're supposed to call it using the name ``Open Sans``, not ``OpenSans``. As a fall-back, I may modify the font-family property with: ``"Open Sans", "OpenSans", "Cantarell", "Helvetica", "Arial", sans-serif``.

I'm not sure if what @matejdro describes is just Firefox not detecting Open Sans, and just falling back to a different font.

I have no idea why it started to act up now, as the referenced line hasn't been changed for a year.

https://github.com/FreshRSS/FreshRSS/blame/edge/p/themes/Origine/origine.css#L89

**Reply by Alkarex** [reply]
We provide the font as a Web font, and it should not pick the OS-local version
https://github.com/FreshRSS/FreshRSS/tree/edge/p/themes/fonts
Debugging welcome

**Comment by math-GH** [comment]
It looks like that you are not the only one with this issue: see #7213
